data/moves.ts

```diff
@@ -19437,13 +19437,20 @@ export const Moves: {[moveid: string]: MoveData} = {
 				}
 				return 5;
 			},
+			onModifyMove(move, source, target) {
+				if (!move.overrideOffensiveStat) return;
```

Comment:
```suggestion
				// This is just for moves that use defensive stats; see below for most of the implementation
				if (!move.overrideOffensiveStat) return;
```

Reply:
That is a bit confusing as all moves use defensive stats. I added a comment that specified that it was for when it used defensive stats as offensive stats